photosynthesis_metrics/__init__.py

```diff
@@ -1,5 +1,5 @@
-from .ssim import ssim, multi_scale_ssim, SSIMLoss, MultiScaleSSIMLoss
-from .msid import MSID, compute_msid
-from .fid import FID, compute_fid
-from .kid import KID, compute_polynomial_mmd
-from .tv import total_variation, TVLoss
+from .ssim import SSIMLoss, MultiScaleSSIMLoss
```

Comment:
I do not like this part at all. It seems like we have only losses but not metrics which is not true